src/test/spec/text-field/text-field.spec.ts

```diff
@@ -518,117 +511,155 @@ describe('TextFieldComponent', function(this: ITestContext) {
     });
   });
 
-  describe('with leading icon', function(this: ITestContext) {
-    beforeEach(function(this: ITestContext) {
+  describe('API - addon-end-always-enabled', function (this: ITestContext) {
```

Comment:
Do we want to add a test that actually attempt to click on an element within this slot when this is enabled?

Reply:
I don't think its strictly needed personally.

Reply:
My only concern is that without it we _could_ have a regression where the classes are still added fine, but the element in this slot ends up being non-interactive.

Reply:
The only reason they were weren't interactable before the css that prevented cursor actions.